source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -181,4 +226,18 @@ content: |
                  extraOptions: extraOptions,
                }
            });
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           const secureClient = new MongoClient(connectionString, {
+           fle_opts = AutoEncryptionOpts( 
+              kms_providers, 
+              key_vault_namespace,
+              schema_map = patientSchema,
+              mongocryptd_bypass_spawn = mongocryptd_bypass_spawn,
+              mongocryptd_spawn_path=mongocryptd_spawn_path 
+           )
+           client = MongoClient( connectionString, auto_encryption_opts = fle_opts )
```

Comment:
Would it be better to stay consistent with spacing for function calls/definitions? Here, there are spaces around the first and last parameters, but in other function calls/definitions, there are no spaces.

Reply:
Fixed:

- Made the spacing similar to other function calls/definition